react/AddProductBtn.tsx

```diff
@@ -32,17 +33,21 @@ class AddProductBtn extends Component<AddProductBtnProps, AddProductBtnState> {
   public render() {
     const { product, large } = this.props
     const { showContent, showLists, isLoading } = this.state
+    
+    const classAddProductBtn = classNames('relative', {
```

Comment:
Same about using this naming, is how we name most of variables like this in other repos.

```suggestion
    const addProductBtnClasses = classNames('relative', {
```